packages/react-liveness/jest.config.js

```diff
@@ -6,6 +6,8 @@ module.exports = {
     '!<rootDir>/**/index.(ts|tsx)',
     // do not collect from top level version and styles files
     '!<rootDir>/src/(styles|version).(ts|tsx)',
+    // do not collect coverage from filed copies from aws-sdk
+    '!<rootDir>/src/components/FaceLivenessDetector/service/utils/CustomWebSocketFetchHandler.ts',
```

Comment:
Thought abt this a bit and have to ask why we would choose to ignore collecting coverage on code we have copied from elsewhere. Hard not to see it as a liability, what do you think @thaddmt

Reply:
+1, are there tests we can copy over from the aws sdk to cover this code?

Reply:
yep i believe you are right, we should be tracking this as we will own this code going forward